modules/gapi/test/internal/gapi_int_pattern_matching_test.cpp

```diff
@@ -9,6 +9,10 @@
 
 #include <stdexcept>
 
+#include <opencv2/gapi/gtransform.hpp>
```

Comment:
need to be removed

Reply:
good job catching this, I forgot to checkout this file to master version - now it should be good